### PR TITLE
chore: update workflows from templates

### PR DESCRIPTION
**Pull request by skjnldsv**
This PR was automatically generated by the workflow dispatcher
Please check that there are no reverted changes before merging ⚠

See templates in https://github.com/nextcloud/.github

### CODE REVIEW

File: .github/workflows/phpunit-sqlite.yml [hunk 1]
```diff
@@ -0,0 +1,143 @@
+# This workflow is provided via the organization template repository
```
Comment (skjnldsv) [inline .github/workflows/phpunit-sqlite.yml:1]: Hum, why is this added? :thinking:

File: .github/workflows/phpunit-mysql.yml [hunk 2]
```diff
@@ -0,0 +1,157 @@
+# This workflow is provided via the organization template repository
```
Comment (skjnldsv) [inline .github/workflows/phpunit-mysql.yml:1]: Same

File: .github/workflows/node.yml [hunk 3]
```diff
@@ -7,6 +7,16 @@ name: Node
 
 on:
   pull_request:
+    paths:
+      - '.github/workflows/**'
+      - 'src/**'
+      - 'appinfo/info.xml'
```
Comment (nickvergessen) [inline .github/workflows/node.yml:13]: same here

File: .github/workflows/lint-eslint.yml [hunk 4]
```diff
@@ -2,15 +2,31 @@
...
+# https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks
+
+name: Lint eslint
+
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/**'
+      - 'src/**'
+      - 'appinfo/info.xml'
```
[the remark this answers is not in the document]
Comment (skjnldsv) [reply .github/workflows/lint-eslint.yml:16]: In the template? We could search for `**/info.xml` or something?

[the remark this answers is not in the document]
Comment (nickvergessen) [reply .github/workflows/lint-eslint.yml:16]: Well server is pretty specific here.
Would go with adjustments in the server

[the remark this answers is not in the document]
Comment (nickvergessen) [reply .github/workflows/lint-eslint.yml:16]: Also the src/ is not correct for server.

**Review by nickvergessen — APPROVED** [review]

**Review by SystemKeeper — APPROVED** [review]
